halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesEditAccountCommand.java

```diff
@@ -296,6 +303,7 @@ protected Account editAccount(KubernetesAccount account) {
     account.setSkin(isSet(skin) ? skin : account.getSkin());
     account.setOnlySpinnakerManaged(isSet(onlySpinnakerManaged) ? onlySpinnakerManaged : account.getOnlySpinnakerManaged());
     account.setCheckPermissionsOnStartup(isSet(checkPermissionsOnStartup) ? checkPermissionsOnStartup : Boolean.TRUE);
+    account.setLiveManifestCalls(isSet(liveManifestCalls) ? liveManifestCalls : Boolean.FALSE);
```

Comment:
I think the false branch of this should be `account.getLiveManifestCalls()` (with a similar pre-existing bug existing in the above line).  Otherwise editing any other field will turn off `liveManifestCalls`.

Reply:
Oh good catch! Fixed the bug in the line above as well.